full-stack-tests/imodelconnection-without-certa/template.env

```diff
@@ -0,0 +1,55 @@
+# All of the configuration variables needed to run the tests are defined and explained here.
+#
+# Only updating the values in this file will not work when attempting to run the tests.
+# Make a copy of this file and rename it to ".env" and that file will automatically be
+# picked up by the tests.
+#
+# WARNING:  Never commit the copied .env file as it will contain passwords or other secrets.
+
+# REQUIRED: Setup the users
+
+# REQUIRED: Setup needed for prefixing the iTwin Platform default url, `https://api.bentley.com`
+## Sets both the environment the tests will be run in and where the iModel(s) are that the tests will run on.
+IMJS_URL_PREFIX=#{TEST_URL_PREFIX}#
```

Comment:
What do you suggest we do? @wgoehrig 
I'm not quite sure what you mean. This test suite might be run in the pipeline but that doesn't mean it can't be used for other debugging purposes. Is there any benefit in removing this .env documentation?